docs/vscode-launch-json.md

```diff
@@ -0,0 +1,125 @@
+# Visual Studio Code - launch.json sample
+
+If you want to debug either the `cluster_manager.py` or the `train_pipeline.py`
+using Visual Studio Code, you can take advantage on this `launch.json` sample
+to run it directly from the IDE.
+
+To use it, you need to fill in all the environment variables
```

Comment:
Alternatively, consider running the shell script <link> and then you can remove all of the `env` sections from the launch.json configuration file.

Reply:
Nice! Added to the tip!